manifoldco/ui
# Resource thing

## Reason for change
Adds a data wrapper for resource details:

```html
<manifold-resource-details resource-id="my-resource" />
```

_Note: this PR also removes our specs, because I realized Catalog was fine to expose, but for our internal stuff it’s good to not make those public_.

## Testing
The auth method is mostly for demo purposes, so it won’t be documented, but here’s how to use it:

1. Log into the Manifold Dashboard (either staging or prod, depending on the `<manifold-connection>` env)
2. Locate the `resource.id` from a resource you’d like to view (you can find this in your **Network** tab—search for `/resources/:id` or something. Place that in `<manifold-resource-details resource-id="my-id" />`.
3. Next, run the following in your browser console: `localStorage.getItem('manifold_api_token')`. Copy the value.
4. In a browser tab showing the UI docs, run the following in a browser console: `localStorage.setItem('manifold_api_token', '[API token from previous step]')`.
5. Refresh, and voila! You should be looking at all the details for that resource.

Note that the `manifold_api_token` is destroyed whenever you log out of Dashboard, or a week has passed and it will expire on its own.

<img width="1214" alt="Screen Shot 2019-04-22 at 12 06 12" src="https://user-images.githubusercontent.com/1369770/56516196-0ed43a00-64f7-11e9-8ed0-bd244a535647.png">
